custom-words.txt

```diff
@@ -14,6 +14,7 @@ CQRS
 CTAP2
 dockerized
 dotfile
+fdroid
```

Comment:
I believe the only reason you had to touch this file was because of your link with "xarantolus/fdroid" as the display text. Can we just use different text? That way it doesn't fill this file with exceptions.